Ocelot.Sample.ApiGateway/Ocelot.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Routes": [
+    {
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "localhost",
+          "Port": 5022
+        },
+        {
+          "Host": "localhost",
+          "Port": 5085
+        }
+      ],
+      "ServiceName": "ConsulDemo",
+      "LoadBalancerOptions": {
+        "Type": "LeastConnection"
+      },
+      "UpstreamPathTemplate": "/hostname",
+      "UpstreamHttpMethod": [ "GET" ],
+      "DownstreamPathTemplate": "/WeatherForecast/hostname",
```

Comment:
Your are only interested by the hostname in your example, why are you copy pasting the WeatherForecast example?